apps/backend/prisma/migrations/20260624180000_attendance_committed_by/migration.sql

```
@@ -0,0 +1,61 @@
+ALTER TABLE "event_attendances"
+ADD COLUMN "committedById" TEXT;
+
+CREATE INDEX "event_attendances_committedById_idx" ON "event_attendances"("committedById");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)